pynest/nest/lib/hl_api_simulation.py

```diff
@@ -460,3 +468,109 @@ def DisableStructuralPlasticity():
 
     """
     sr('DisableStructuralPlasticity')
+
+@check_stack
+def Dump(selections: List = ("nodes", "synapses")) -> Dict:
+    """Returns a dictionary containing the current net in memory for serialization.
+
+    Only a subset of the data can be serialized.
+
+
+    Parameters
+    ----------
+    selections : List[strings], optional
+        Only obtain a subset of the net parameters. Options are "nodes" and/or "synapses"
+
+
+    Returns
+    -------
+    Dict:
+        Object containing the specified serialization sorted by dict key likes "nodes" or "synapses"
+
+
+    See Also
+    ---------
+
+    Load : Function to construct the net from the data.
+
+    """
+
+    sli_func("({}) M_WARNING message".format("Dumping is currently only supported for a single process."))
+    dumpdata = dict()
+    numnetwork = GetKernelStatus("network_size")
+
+    # nothing added
+    if numnetwork < 2:
```

Comment:
I think a more explanatory comment would help here... I've no idea where this magic two comes from...

Reply:
it seems that an empty network already has network size 1. but I agree that this should be added in a comment.